qiskit/providers/fake_provider/generic_backend_v2.py

```diff
@@ -53,15 +53,15 @@
     "cz": (1e-8, 9e-7, 1e-5, 5e-3),
     "id": (3e-8, 4e-8, 9e-5, 1e-4),
     "rz": (0.0, 0.0),
-    "sx": (1e-8, 9e-7, 1e-5, 5e-3),
-    "x": (1e-8, 9e-7, 1e-5, 5e-3),
+    "sx": (3e-8, 4e-8, 9e-5, 1e-4),
+    "x": (3e-8, 4e-8, 9e-5, 1e-4),
```

Comment:
Strictly this still lets `cz` have a lower minimum bound than `sx`.  I imagine that this doesn't come up much in practice because the duration intervals are sampled from the uniform distribution rather than the logarithmic one, but it still feels odd.

Reply:
that is a good point, I have modified the ranges (using real eagle, torino and falcon devices as reference) to avoid the overlap between 1q and 2q, the new ranges are:

- 1q gates: 30 - 60 ns
- 2q gates: 80 - 970 ns
- measure: 700 - 1500 ns

I am considering lowering the upper bound of the 2q gates to ~700 ns to also avoid the overlap with measure (the range in real devices is just really broad), although I don't think this overlap affects the specific case of DD.

Reply:
I think it would be important not to mix different architectures (heron vs. eagle for example) and use time ranges from a single device 🙂